jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._marks = {}
         self._dict = {}  # dictionary representation of current mouseover info
+        self._spectral_axis_index = 2  # Needed for cube data
```

Comment:
will this choke if multiple cubes use different values for `spectral_axis_index`?

Reply:
Actually, this gets set based on the reference data cube when it's actually used, it should be fine.